docs/documents/faq.md

```diff
@@ -0,0 +1,38 @@
+# FAQ by Document Type
+
+> Find answers to document specific questions
+
+## Overview
+
+In a perfect world, everybody might be adept at producing and consuming OSCAL. In reality, human-readable documentation is still a necessity and each vendor represents their content differently. Consequently, it can be challenging to fit existing documentation into the OSCAL model while still maintaining consistent outputs.
+
+This FAQ will help you navigate common questions and issues related to specific documents.
+
+## FedRAMP
+
+### SSP
+
+::: details How do I populate "Nature of Agreement" found in Leveraged Authorizations and Interconnections tables?
```

Comment:
Worth adding "... found in Section #"?

Reply:
great idea

Reply:
Get a bit tricky with Rev4 and Rev5. And the question is for two different sections.

Reply:
How's that for now?

<img width="740" alt="image" src="https://github.com/paramify/support/assets/24945085/983eb077-aec9-4806-b100-9cac79fc3cae">